### PR TITLE
Make a comment more inclusive

### PR DESCRIPTION
**Pull request by Ladyprowess**
Rather than "master" use "controller"

- [x] Make sure you are opening from a *topic/feature/bugfix branch* (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
~Ensure you have provided tests - that demonstrate the feature works or fixes the issue~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -245,7 +245,7 @@ If you wanted to see if a new version of a library works with blue ocean:
 * Check back later for build success (ie unit tests)
 * The Acceptance Test Harness will normally be automatically triggered after a successful PR build, however, it ie best to check it has run: (https://ci.blueocean.io/job/ATH-Jenkinsfile/job/master/) - consult a blue ocean contributor (see below) and they will ensure it has run. This is required for a dependency change.
 * Contact a contributor (see below) to let them know of your proposed change so they can review it and do extra testing
-* Ensure any dependencies are released to the non beta UC, before merging to master when approved. 
+* Ensure any dependencies are released to the non beta UC, before merging to controller when approved.
```
Comment (olamy) [inline README.md:248]: the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

File: README.md [hunk 2]
```diff
@@ -266,7 +266,7 @@ The Acceptance test suite is located here: [acceptance-tests](./acceptance-tests
 When the ATH passes and there is a consensus that a release can be performed: 
 
 * Ensure that the person doing the release has permissions for all the blueocean modules here: https://github.com/jenkins-infra/repository-permissions-updater/tree/master/permissions (or it will fail)
-* Switch to the branch to release from (usually master)
+* Switch to the branch to release from (usually controller)
```
Comment (olamy) [inline README.md:269]: the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

**Review by olamy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ladyprowess) [reply README.md:248]: > the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

Okay, noted... Will resolve that

File: README.md (hunk 2 above)
Comment (Ladyprowess) [reply README.md:269]: > the code use the canonical branch called `master` this has nothing to do with  a `controller` so it should be `main`

Okay, will resolve that

**Pull request closed without merging**